sympy/core/tests/test_args.py

```diff
@@ -639,6 +639,11 @@ def test_sympy__core__function__Derivative():
     from sympy.core.function import Derivative
     assert _test_args(Derivative(2, x, y, 3))
 
+def test_sympy__core__function__AppliedFuncDerivative():
+    from sympy.core.function import AppliedFuncDerivative
+    f = Function('f')
+    assert _test_args(AppliedFuncDerivative(f(x**2, y), S(1), S(2), S(2)))
```

Comment:
It seems like this should be tested without wrapping expected ints in `S()`.

Reply:
Actually, this class was part of an idea which I solved through another way. Its not needed anymore and I will remove it in the next commit.